### PR TITLE
Fixed the bug that caused the program to crash.

### PR DESCRIPTION
**Pull request by chenbinn**
run examples/notification exec, when recieve notification data more then 2 times case the program to crash.

### CODE REVIEW

File: dbus/gattlib_notification.c [hunk 1]
```diff
@@ -104,8 +104,6 @@ static gboolean on_handle_characteristic_property_change(
 				}
 			}
 
-			if (key != NULL)
-				g_free(key);
```
Comment (oliviermartin) [inline dbus/gattlib_notification.c:108]: With your change, we leak some memory every time we receive a notification, no?

**Comment by kdewald** [comment]
This fix is incorrect. I submitted a proper fix in: https://github.com/labapart/gattlib/pull/215

**Comment by chenbinn** [comment]
> This fix is incorrect. I submitted a proper fix in: #215
g_free(key);   this code cause crash.

**Comment by chenbinn** [comment]
> This fix is incorrect. I submitted a proper fix in: #215

g_free(key); t
his code cause crash.

**Pull request closed without merging**